archinstall/lib/models/disk_encryption.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass, field
+from enum import Enum, auto
+from typing import Optional, List, Dict, TYPE_CHECKING, Any
+
+from archinstall.lib.hsm.fido import Fido2Device
```

Comment:
@svartkanin can we some how configure VSCode to not behave like this? Because VSCode really screw `archinstall` over as a library when I miss these and they get merged in.

Reply:
Yeah sorry about that, it's my Pycharm that's messing it up. I'll have a look at the settings to see if things can be tweaked

Reply:
@torxed what exactly is the issue that is happening with absolute paths btw? 
Maybe we can see if this could be fixed differently as well. 
There are some good arguments for using absolute paths in general https://softwareengineering.stackexchange.com/questions/159503/whats-wrong-with-relative-imports-in-python/159505#159505 :) but this doesn't mean I'm necessarily in favor of picking one over the other:)